go/teams/errors.go

```diff
@@ -230,9 +230,13 @@ func IsTeamReadError(err error) bool {
 		switch keybase1.StatusCode(e.Code) {
 		case keybase1.StatusCode_SCTeamReadError:
 			return true
+		default:
```

Comment:
think this would be more readable as:
```
aerr, ok := err.(libkb.AppStatusError)
return ok && aerr.Code == keybase1.StatusCode_SCTeamReadError
```